sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -6590,6 +6590,7 @@ private void SetNameFieldsIfNull()
         /// <remarks>
         /// A <see cref="Exception"/> will be thrown if a failure occurs.
         /// </remarks>
+        [PrivilegedOperation]
         public virtual Uri GenerateSasUri(BlobSasPermissions permissions, DateTimeOffset expiresOn) =>
```

Comment:
What's the criteria for identifying what is, or may potentially be, a privileged operation?   Is the intent to flag just obvious low-level operations or try to decorate any higher level abstractions that may use them - directly or transitively?   My concern for this is that we may end up decorating a large surface area or missing something in the call chain.

Reply:
"Something that should be audited."  We're in the process of defining that and came to an agreement with the Storage team and Core Azure folks on this set of APIs to get started.  It's part of the beauty of using an internal shared source attribute - we're not making any public API change and have flexibility to iterate on this in partnership with the Core folks.  At the moment there's no plan for transitivity, but we don't have so many convenience APIs that should be a problem in the short term (i.e., this PR is already transitively closed over SAS generation).